src/index.ts

```diff
@@ -44,4 +44,3 @@ export * from './Upload';
 
 // Utility functions
 export * from './utils/formInput';
```

Comment:
should we also move formInput to the js-utils lib in course of the refactoring?

Reply:
It is not as generically useful, its output types are coupled to the Select component. I moved it in there to make it coherent and eliminate the generic `utils` directory.